apps/portals/src/portal-components/nf/BrowseToolsPage.tsx

```diff
@@ -71,39 +83,47 @@ const BrowseToolsPage = () => {
           Drill-down to explore specific types of NF research tools.
         </Typography>
         <div className="categories">
-          <button onClick={() => gotoExploreToolsWithSelectedResource('Animal Model')}>
+          <button
+            onClick={() => gotoExploreToolsWithSelectedResource('Animal Model')}
+          >
             <AnimalModels />
-            <Typography variant="headline3">
-              Animal Models
-            </Typography>
+            <Typography variant="headline3">Animal Models</Typography>
           </button>
-          <button onClick={() => gotoExploreToolsWithSelectedResource('Antibody')}>
+          <button
+            onClick={() => gotoExploreToolsWithSelectedResource('Antibody')}
+          >
             <Antibodies />
-            <Typography variant="headline3">
-              Antibodies
-            </Typography>
+            <Typography variant="headline3">Antibodies</Typography>
           </button>
-          <button onClick={() => gotoExploreToolsWithSelectedResource('Genetic Reagent')}>
+          <button
+            onClick={() =>
+              gotoExploreToolsWithSelectedResource('Genetic Reagent')
+            }
+          >
             <PlasmidsReagents />
-            <Typography variant="headline3">
-              Genetic Reagents
-            </Typography>
+            <Typography variant="headline3">Genetic Reagents</Typography>
           </button>
-          <button onClick={() => gotoExploreToolsWithSelectedResource('Cell Line')}>
+          <button
+            onClick={() => gotoExploreToolsWithSelectedResource('Cell Line')}
+          >
             <CellLines />
-            <Typography variant="headline3">
-              Cell Lines
-            </Typography>
+            <Typography variant="headline3">Cell Lines</Typography>
           </button>
-          <button onClick={() => gotoExploreToolsWithSelectedResource('Biobank')}>
+          <button
+            onClick={() => gotoExploreToolsWithSelectedResource('Biobank')}
+          >
             <Biobanks />
-            <Typography variant="headline3">
-              Biobanks
-            </Typography>
+            <Typography variant="headline3">Biobanks</Typography>
           </button>
         </div>
         <div className="center-content bootstrap-4-backport">
-          <Button className="pill-xl" variant="primary" onClick={() => gotoExploreTools()}>View All Tools</Button>
+          <Button
+            className="btn-wide"
+            variant="primary"
+            onClick={() => gotoExploreTools()}
+          >
+            View All Tools
+          </Button>
```

Comment:
Rename `pill-xl` class to `btn-wide` to maintain styling, but remove reference to `pill`